core/storage/user/gae_models.py

```diff
@@ -615,21 +615,23 @@ def create(cls, user_id, story_id):
             id=instance_id, user_id=user_id, story_id=story_id)
 
     @classmethod
-    def get(cls, user_id, story_id):
+    def get(cls, user_id, story_id, strict=False):
```

Comment:
By default, strict should be True, as it is in the base storage model.

Reply:
Done.